app/scripts/lib/app-start.js

```diff
@@ -636,6 +636,7 @@ define(function (require, exports, module) {
 
     _getErrorPage: function (err) {
       if (OAuthErrors.is(err, 'MISSING_PARAMETER') ||
+          OAuthErrors.is(err, 'INVALID_PARAMETER') ||
```

Comment:
@vbudhram - I added this line. We didn't throw any INVALID_PARAMETER errors previously, so this needed to be added.